tests/test_db_store.py

```diff
@@ -38,18 +39,32 @@ def test_retrieve_soil_moisture(self):
         mock_cursor = mock.Mock()
         store = db_store.SoilMoistureStore(mock_cursor)
         mock_cursor.fetchall.return_value = [
-            ('2016-07-23 10:51:09.928000', 300),
-            ('2016-07-23 10:52:09.928000', 400)
+            ('2016-07-23 10:51:09.928000-05:00', 300),
+            ('2016-07-23 10:52:09.928000-05:00', 400)
         ]
         soil_moisture_data = store.retrieve_soil_moisture()
         soil_moisture_data.sort(
             key=lambda SoilMoistureRecord: SoilMoistureRecord.timestamp)
 
-        self.assertEqual(soil_moisture_data[0].timestamp,
-                         '2016-07-23 10:51:09.928000')
+        self.assertEqual(
+            soil_moisture_data[0].timestamp,
+            datetime.datetime(
+                2016,
+                7, 23,
+                10,
+                51,
+                9, 928000,
+                tzinfo=tzoffset(None, -18000)))
```

Comment:
I wish we had a better solution for this. I feel like it's hard for the reader to recognize that `-05:00` matches up to `tzoffset(None, -18000)`.

I have two ideas, but I don't love either of them.

### 1. Use UTC

We can change the `-05:00` part to `+00:00` (UTC) or `Z` (shorthand for UTC), then in the assertion, change it to `tzinfo.pytz.utc`. It's a little easier to identify, but it obscures the fact that the database stores times in local time instead of UTC.

### 2. Use helper function for time zone

We can alternatively replace the `tzoffset` call with a global constant like:

```python
# Timezone offset info for EST (UTC minus 5 hours).
UTC_MINUS_5 = tzoffset(None, -18000)
...

        self.assertEqual(
            soil_moisture_data[1].timestamp,
            datetime.datetime(
                2016,
                7, 23,
                10,
                52,
                9, 928000,
                tzinfo=UTC_MINUS_5))
```

What do you think?